__manifest__.py

```diff
@@ -13,6 +13,7 @@
         "views/openacademy_course_views.xml",
         "views/openacademy_session_views.xml",
         "views/res_partner_views.xml",
+        "wizards/openacademy_wizard_views.xml",
```

Comment:
Same case that above.

```suggestion
        "wizard/openacademy_wizard_views.xml",
```